doc/configuration.rst

```diff
@@ -728,6 +728,31 @@ created with ID 2". We can view the details of this new investigator by entering
 	created  2015-09-09 09:53:28.989481 -0400 EDT
 	modified 2015-09-09 09:53:28.989481 -0400 EDT
 
+Recent versions of MIG have introduced the concept of investigators that can be
```

Comment:
I wouldn't bother with "recent".
`MIG supports two levels of access its users: investigators and administrators.`

By the way, should administrators be allowed to launch investigations, or are the two roles mutually exclusive?